worker/pom.xml

```diff
@@ -72,6 +72,11 @@
             <artifactId>logback-classic</artifactId>
             <version>${logback.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-jaxb-annotations</artifactId>
+        </dependency>
```

Comment:
Had to add this dependency explicitly to get a successful `mvn package` - I think *logstash* needs this dependency.